docs/package.json

```
@@ -8,7 +8,8 @@
     "preview": "ROLLDOWN_OPTIONS_VALIDATION=loose vitepress preview"
```

We can probably drop the env variable here too then!
```suggestion
    "preview": "vitepress preview"
```